### PR TITLE
Add TOS Link from Config

### PR DESCRIPTION
**Pull request by amy-corson-ibigroup**
**Description:**

Allows a URL override for the Terms of Service that will direct to an external link 

**PR Checklist:**
- [n/a] Does the code follow accessibility standards (WCAG 2.1 AA Compliant)?
- [n/a] Are all languages supported (Internationalization/Localization)?
- [x] Are appropriate Typescript types implemented?

### CODE REVIEW

File: lib/components/user/terms-of-use-pane.tsx [hunk 1]
```diff
@@ -16,11 +16,15 @@ const TermsOfUsePane = ({
   disableCheckTerms,
   handleBlur,
   handleChange,
+  locale,
+  termsOfServiceLink,
```
Comment (binh-dam-ibigroup) [inline lib/components/user/terms-of-use-pane.tsx:20]: Please add `termsOfServiceLink` to the config types. I guess `termsOfStorageSet` needs to be added too.

File: lib/components/user/terms-of-use-pane.tsx [hunk 2]
```diff
@@ -99,6 +103,8 @@ const TermsOfUsePane = ({
 }
 const mapStateToProps = (state: any) => {
```
Comment (binh-dam-ibigroup) [inline lib/components/user/terms-of-use-pane.tsx:104]: Use type `AppReduxState`.

**Review by binh-dam-ibigroup — CHANGES_REQUESTED** [review]
Could you please update the config types? Looks good otherwise.

**Review by AdrianaCeric — APPROVED** [review]
Great work!

**Pull request merged**